src/normaliz.cc

```
@@ -350,8 +486,15 @@ static Obj _NmzConeIntern(Obj input_list)
             return Fail;
         }
         string type_str(CSTR_STRING(type));
-
         Obj M = ELM_PLIST(input_list, i+2);
+        if(type_str.compare("polynomial") == 0){
+            if(!IS_STRING_REP(M)){
+                cerr << "Element " << i+2 << " of the input list must be a string" << endl;
+            }
+            polynomial = string(CSTR_STRING(M));
```

Uhm... so if we detect that `M` isn't a string, we print an error... and then access it as if it was a string anyway? Shouldn't we instead throw an exception?